package.json

```diff
@@ -1,44 +1,25 @@
 {
-  "name": "structurize",
-  "version": "1.4.1",
-  "description": "Text format hinting library for estimating the format of a given input string",
-  "main": "index.js",
-  "scripts": {
-    "test": "node_modules/mocha/bin/mocha"
-  },
-  "repository": {
-    "type": "git",
-    "url": "https://github.com/avinoamr/structurize.git"
-  },
-  "keywords": [
-    "text",
-    "format",
-    "hint",
-    "guess",
-    "estimate",
-    "parse",
-    "structure",
-    "stream",
-    "json",
-    "json5",
-    "csv",
-    "tsv",
-    "querystring"
-  ],
-  "author": "Roi Avinoam <avinoamr@gmail.com>",
-  "license": "MIT",
-  "bugs": {
-    "url": "https://github.com/avinoamr/structurize/issues"
-  },
-  "homepage": "https://github.com/avinoamr/structurize",
-  "devDependencies": {
-    "csv-parse": "^1.0.4",
-    "jsonstream2": "^1.1.1",
-    "mocha": "^2.3.3",
-    "tar": "^2.2.1",
-    "xlsx": "^0.8.2"
-  },
-  "dependencies": {
-    "overflow": "1.1.4"
-  }
+    "name": "structurize",
+    "version": "1.4.2",
+    "description": "Text format hinting library for estimating the format of a given input string",
+    "author": "Roi Avinoam <avinoamr@gmail.com>",
```

Comment:
please change the author to panoply.io

Reply:
Will this change meet [MIT license](https://github.com/panoplyio/structurize/blob/master/LICENSE) limitations and conditions?

Reply:
Yeah, but it's public repo and the author of this lib is indeed Roi Avinoam,